packages/shared/src/lib/flows/actions/action.ts

```diff
@@ -24,7 +24,7 @@ const commonActionProps = {
   }),
   name: Type.String({
     description:
-      'Name of the step (action), used for reference in code or configurations. Usually the name is in the format step_index',
+      'Name of the step (action), used for reference in other step configurations. The name should always be in the format: "step_{autoincrement}", example: "step_1", "step_2", etc.',
```

Comment:
@alexandrudanpop do we need this? step_{autoincrement} ? it can absolutely be {{step_stepId}}. Can also be {{my_step}}

Reply:
fixed, the tool will do it.